components/BotForm.tsx

```diff
@@ -387,29 +394,26 @@ export const BotForm = () => {
                   )
                   .map((item: any) => item.current_asset_1_reserves_in_usd)
               );
-              if (assetRates[assetId]) {
-                const amountToTrade =
-                  orderAlgoDepth * assetRates[assetId].price * ladderTiers;
-                if (maxLiquidity) {
-                  if (amountToTrade >= maxLiquidity * 0.5) {
-                    setASAError(
-                      "This ASA‘s liquidity is too high and risky to use this bot"
-                    );
-                    return true;
-                  } else if (amountToTrade > maxLiquidity * 0.1) {
-                    setASAWarning("Warning, this ASA‘s liquidity is high");
-                    return false;
-                  }
-                } else {
+              const amountToTrade = ladderTiers * algoRate * orderAlgoDepth;
+              if (maxLiquidity) {
+                if (amountToTrade >= maxLiquidity * 0.5) {
                   setASAError(
-                    "This ASA‘s liquidity is too low on Tinyman to use this bot"
+                    "This ASA‘s liquidity is too high and risky to use this bot"
```

Comment:
The liquidity should say low, not high. High liquidity is a good thing

Reply:
We had a conversation on this some days back;

You said: 
Less than 10% is good
Above 10% give warning
Above 50% is risky and should not trade on the bot